bin/assume-aws-role.js

```diff
@@ -63,6 +63,13 @@ if (command === "add") {
   process.exit(0);
 }
 
+if (command == "list") {
+	var config = readConfig(filename);
+	var aliases = Object.keys(config);
+  	console.error("Defined aliases: %s" , aliases);
```

Comment:
I think using `console.log()` and `process.exit(0)` is more accurate here because you're requesting a list of the aliases